executor/distsql.go

```diff
@@ -407,11 +417,14 @@ func (e *IndexLookUpExecutor) open(ctx context.Context, kvRanges []kv.KeyRange)
 			return errors.Trace(err)
 		}
 	}
+	return nil
+}
 
+func (e *IndexLookUpExecutor) startWorkers(ctx context.Context) (err error) {
```

Comment:
would it impact the latency performance after we lazily start works when `Next()` is called?

Reply:
Yes, it will.
But is it weird to make it run at `Open()`?
In my mind, `Open()` is just used to prepare and reserve resource.
So it shouldn't start to work at that time?
And we have already implemented lazy start strategy in many other executors like `Projection`, `Sort`, `HashJoin`, `MergeJoin`.
So it's better to implement `IndexLookUp` in the same way?